pypeit/core/telluric.py

```diff
@@ -307,55 +307,57 @@ def eval_telluric(theta_tell, tell_dict, ind_lower=None, ind_upper=None):
           humidity, airmass)
 
        2. convolution of the atmosphere model to the resolution set by
-          resln.
+          the spectral resolution.
 
-       3. Optionally, if len(theta_tell) == 7, application of a shift and
-          a stretch to telluric model. If len(theta_tell) == 5, no shift is applied
-          and only the first two steps are performed.
+       3. (Optional) shift and stretch the telluric model.
```

Comment:
Please make it more clear that the shift and stretch correction are omitted if len(theta_tell) = 5. Please avoid truncating documentation, it just makes things less clear.

Reply:
I wasn't sure, so I was just explaining what I thought was the intent of the code.  I agree that we don't want the docs explaining something that isn't true.  But I think it's true, within the context of this function, that theta_tell can have either 5 or 7 elements.  It's arguably a different question whether or not it's ever used with 5 elements.

Reply:
Yes that is fine, we can leave it as is.